### DISCUSSION TITLE
attachment deletion from file system

### DISCUSSION THREAD

**Discussion by McDiggy69** [Leantime/leantime#715] [General]
In version 2.1.8 attachments stored on the file system were not deleted when projects they are associated with were deleted.  I see nothing in the version 2.1.9 release notes about fixing this problem.

Has the problem been fixed in version 2.1.9?  If not, any plans to address the problem in the next release?

**Comment by marcelfolaron** [comment]
Thanks for pointing that out. It has been resolved for individual file deletion but I realized this is not part of the project deletion. I will make this discussion an issue
